projects/magento/magentoCI.config.cjs

```diff
@@ -37,7 +37,9 @@ const config = {
     actionTimeout: 0,
 
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: process.env.MAGENTO_BASE_URL,
+    // baseURL: process.env.MAGENTO_BASE_URL,
+    baseURL: "http://magento2.test.com",
```

Comment:
Ops, we can't actually do this. It will break this repository's actions. :(

Reply:
I won't break the actions, since we use another config file for the repo itself.

Reply:
Can we remove the commented code either way?

Reply:
Done!